app/models/cuisine.rb

```diff
@@ -0,0 +1,20 @@
+# == Schema Information
+#
+# Table name: cuisines
+#
+#  id         :integer          not null, primary key
+#  name       :string
+#  icon       :string
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+
+class Cuisine < ApplicationRecord
+
+  validates_presence_of :name
+
+  validates_length_of :name, :maximum => 70
+
+  has_and_belongs_to_many :restaurants
```

Comment:
will a restaurant have multiple cuisines? otherwise `belongs_to` should be enough

[has_and_belongs_to_many](http://guides.rubyonrails.org/association_basics.html#the-has-and-belongs-to-many-association) is a many-to-many relationship and creates another database table that slows down queries.